test/presubmit-integration-tests-previous-release.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
I don't really get this file (but could be also my ignorance ;)

* Shouldn't it be called '...latest-release.sh' as it refers to the latest release whereas "latest" refers to HEAD of master (unreleased code). A bit confusing. Or maybe call it really with the version number ('presubmit-integration-tests-0.6.0.sh') and update the configuration that calls this file
* Who is calling this file ?
* Why it is calling presubmit checks and not the e2e tests like in e2e-tests.sh  but with a environment variable set ?

Reply:
So I would have expected something like

```
export KNATIVE_VERSION="0.6.0"
$(dirname $0)/e2e-tests.sh
```

Reply:
Talked offline with @houshengbo . It's complicated (as the second job needs to be a custom job).

Reply:
Thank @rhuss for his suggestion. We will use latest-release.
The repo test-infra defines four basic type of jobs, every PR can enable: integration-tests, build-tests, go-coverage, and unit-tests. We can only four regular jobs, with one for each of them respectively. It means even if we would like to have a new INTEGRATION TEST, we need to go through CUSTOM TEST JOB in test-infra. We cannot use the existing `presubmit-tests.sh --integration-tests` with more argument to specify the version or anything else. That was the reason we need to have a separate script like this one, with the correct env var setting for a specific version, to launch separately by the CUSTOM TEST JOB in test-infra.